examples/vitals/tesla.proto

```diff
@@ -80,19 +363,162 @@ message Device {
     optional google.protobuf.Timestamp firstCommunicationTime = 8;
     optional google.protobuf.Timestamp lastCommunicationTime = 9;
     optional ConnectionParameters connectionParameters = 10;
-    repeated DeviceAttributes deviceAttributes = 11;
+    optional DeviceAttributes deviceAttributes = 11;
 }
 
 message SiteControllerConnectedDevice {
     optional Device device = 1;
 }
 
 message SiteControllerConnectedDeviceWithVitals {
-    repeated SiteControllerConnectedDevice device = 1;
+    SiteControllerConnectedDevice device = 1;
```

Comment:
Breaking change. I had this wrong in my initial attempt.

Reply:
Makes sense.  For all of these updates, for pypowerwall, I think we can accomodate without major API impact to the pypowerwall functions (as minor rev) but since it could impact vitals() output, I'll increment as a major version rev. to signal caution.